examples/guess-the-sketch/matchmaker/frontend_k8s.yaml

```diff
@@ -12,6 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: frontend-sa
+  annotations:
+    iam.gke.io/gcp-service-account: om-caller@peteryizhong-gke-dev.iam.gserviceaccount.com
```

Comment:
Same concern here for the service account.

Reply:
Same as the reply in the other comment